scripts/build.sh

```diff
@@ -21,11 +25,19 @@ babel --env-name esmBrowser src --source-root src --out-dir "$DIR/esm-browser" -
 babel --env-name esmNode src --source-root src --out-dir "$DIR/esm-node" --copy-files --quiet
 
 # No need to have the CLI files in the esm build
+rm -rf "$DIR/commonjs-browser/bin"
+rm -rf "$DIR/commonjs-browser/uuid-bin.js"
 rm -rf "$DIR/esm-browser/bin"
 rm -rf "$DIR/esm-browser/uuid-bin.js"
 rm -rf "$DIR/esm-node/bin"
 rm -rf "$DIR/esm-node/uuid-bin.js"
 
+for FILE in "$DIR"/commonjs-browser/*-browser.js
+do
+    echo "Replacing node-specific file for commonjs-browser: $FILE"
+    mv "$FILE" "${FILE/-browser.js/.js}"
+done
+
```

Comment:
Can you add a comment here explaining *why* we do this?

Reply:
Added a longer description of what we're doing here.